Packs/PrismaCloud/Integrations/PrismaCloudIAM/PrismaCloudIAM.py

```diff
@@ -168,24 +156,21 @@ def get_app_fields(self) -> Dict[str, Any]:
         :return: The user schema fields dictionary
         :rtype: ``Dict[str, str]``
         """
-
         return USER_SCHEMA
 
-    def create_login(self):
-        try:
-            uri = '/login/'
-            body = {
-                "username": self._username,
-                "password": self._password,
-                "customerName": self._customer_name
-            }
-            return self._http_request(
-                method='POST',
-                url_suffix=uri,
-                json_data=body
-            )
-        except DemistoException as e:
-            return_error(e.message)
+    def login(self):
```

Comment:
I think that it would be best to catch the exception and return a clear message, such as "failed to login...".

Reply:
i don't print a new custom message because the current message from the `e.message`
are `Error in API call [401] - Unauthorized {"message": "invalid_credentials"} (85)`
so from my point of view it's clear enough